.github/workflows/run-mocha-tests.yml

```
@@ -11,11 +11,11 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ["ubuntu-latest", "windows-2022"]
```

actionlint (1.7.12)

[error] label "windows-2022" conflicts with label "ubuntu-latest" defined at line:14,col:15. note: to run your job on each workers, use matrix

(runner-label)